spark-tsfile/src/main/scala/org/apache/iotdb/tsfile/Transformer.scala

```diff
@@ -0,0 +1,174 @@
+/**
+  * Licensed to the Apache Software Foundation (ASF) under one
+  * or more contributor license agreements.  See the NOTICE file
+  * distributed with this work for additional information
+  * regarding copyright ownership.  The ASF licenses this file
+  * to you under the Apache License, Version 2.0 (the
+  * "License"); you may not use this file except in compliance
+  * with the License.  You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing,
+  * software distributed under the License is distributed on an
+  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  * KIND, either express or implied.  See the License for the
+  * specific language governing permissions and limitations
+  * under the License.
+  */
+package org.apache.iotdb.tsfile
+
+import org.apache.iotdb.tsfile.common.constant.QueryConstant
+import org.apache.spark.sql.{SparkSession, _}
+import org.apache.spark.sql.types._
+
+
+object Transformer {
```

Comment:
Since you can read dataframe in two forms, this transformer is needless.

Reply:
I think Transformer has 2 function:
1. sometimes we get wide table and then need to transform to narrow table, transformer can be use in this condition.
2. transformer support narrow table to wide table, which can be write into tsfile

Reply:
That reminds me... We need to support writing new form to tsfile, which is easier than old form.